x-pack/plugin/repository-encrypted/build.gradle

```diff
@@ -18,9 +18,9 @@ dependencies {
     // required for integ tests of encrypted FS repository
     internalClusterTestImplementation project(":test:framework")
     // required for integ tests of encrypted cloud repositories
-    internalClusterTestImplementation project(path: ':plugins:repository-gcs', configuration: 'internalClusterTestArtifacts')
-    internalClusterTestImplementation project(path: ':plugins:repository-azure', configuration: 'internalClusterTestArtifacts')
-    internalClusterTestImplementation(project(path: ':plugins:repository-s3', configuration: 'internalClusterTestArtifacts')) {
+    internalClusterTestImplementation testArtifact(project(':plugins:repository-gcs'), 'internalClusterTest')
+    internalClusterTestImplementation testArtifact(project(':plugins:repository-azure'), 'internalClusterTest')
+    internalClusterTestImplementation(testArtifact(project(':plugins:repository-s3'), 'internalClusterTest')) {
```

Comment:
nit: can we remove the parens here to be consistent. This is making my brain hurt.

Reply:
no we can't as there's a configuration block is following. that doesn't work without the parenthesis due to groovy not supporting this syntax

Reply:
Ah, my bad, I didn't see that. Ok, this is fine with me then.